### DISCUSSION TITLE
Problems on lineage-14.1 20181113 UNOFFICIAL for nx591j

### DISCUSSION THREAD

**Discussion by sullennv** [kdrag0n/safetynet-fix#182] [Q&A]
Hi! I am using lineage 14.1 20181113 UNOFFICIAL for nx591j based on Android 7.1.2 and install safetynet fix 2.2.1 for Magisk v24.3 and get the error message "this version is for Android 8 and above".
I try to install safetynet fix v2.2.0 and get critical errors of google services.
Please tell me how to solve this problem?

**Comment by asverq** [comment]
I have the same problem on the android 7.1.2, how i understand safetynet-fix has no supporting the android version anymore
